src/Webform.js

```diff
@@ -998,7 +998,9 @@ export default class Webform extends NestedDataComponent {
             "submitButton",
             (options) => {
                 this.submit(false, options).catch((e) => {
-                    options.instance.loading = false;
+                    if (options?.instance) {
+                        options.instance.loading = false;
+                    }
```

Comment:
better use of optional chaining
```suggestion
                    options?.instance?.loading = false
```